### ISSUE TITLE
[Proposal] Implement Windows Mail's visual treatment for unread emails

### ISSUE THREAD

**Issue by MilkTeaCat52** [bkaankose/Wino-Mail#198]
**Describe the solution you'd like**
Wino Mail's visual treatment for unread emails differs from Windows Mail. In Windows Mail, there is an accented vertical bar indicating an unread email. The timestamp is accented as well. These make it more obvious at a glance which emails are unread. It would be nice if the visual treatment in Wino could be changed to match Windows Mail, or for there to be an option to choose this visual treatment. 

**Is your proposal implemented in Windows Mail?**
Yes

**Additional context**
![image](https://github.com/bkaankose/Wino-Mail/assets/91018175/8eb7072c-0836-43d2-a57f-1c144ad2278f)

**Comment by bkaankose** [comment]
Accented vertical bar is reserved for selected item in Windows 11 design guidelines. That's not the case for WMC. If Wino had that, it'll be 2 accented bars for Wino that runs on Windows 11. 

Personally I find current implementation clear on eyes and easy to follow. There were also complains about the same thing you mentioned here before. However, I am open for the idea of more personalized visuals for mail list.

WMC and Wino will differ from each other at some point. If there is a high demand on this matter I can prioritize it but looks like this is a minor UI thing for now.

**Comment by MilkTeaCat52** [comment]
I agree that having two accented bars would look cluttered. WMC and Outlook get around this by not having an accented bar for selection in the mail pane, while keeping it in the navigation pane. I think this is a pretty good approach and succeeds in keeping the UI clean.

The current implementation in Wino seems kind of empty coming from WMC, though it might just be because I'm so used to WMC and Outlook. I do think there are some advantages to an accented bar and time, though. It would make the color on the email more even and would help keep some separation between emails.

I get that UI changes aren't as pressing as feature completeness. However, I hope at some point, there will be an option to forego the selection accented bar and have it for unread emails instead. Personalization is always a plus!

**Comment by Tiktack** [comment]
Completely agree with @bkaankose that current version looks clear and doesn't require changes. + basically it follows fluent UI. Selected item has vertical bar.

**Comment by bkaankose** [comment]
There is a good amount of request on this matter. I'm open for some ideas, but having 2 vertical bar is not one of them for sure. Would be great if designer folks can jump in for some feedback to make unread mails more clear for everybody.

**Comment by todordvv** [comment]
I really struggle to distinguish unread from read emails on dark theme. Maybe a bullet point or some other indicator beside the SUBJECT of the email in the list could work? Something like this:

![Image](https://github.com/user-attachments/assets/4ad74718-16e7-49c7-bda1-ff65310c5c7d)

**Comment by bkaankose** [comment]
This issue has been fixed with v1.10.3.

New visual for unread mails look like this
![Image](https://github.com/user-attachments/assets/76ed5286-2db9-41d3-a606-a072551a77cd)

**Comment by todordvv** [comment]
I noticed this! Thank you so much!!!!

Immensely better now!
